plugins/bkpr/bookkeeper.c

```diff
@@ -414,58 +472,80 @@ static struct command_result *json_list_account_events(struct command *cmd,
 
 	res = jsonrpc_stream_success(cmd);
 	json_array_start(res, "events");
-	for (size_t i = 0, j = 0, k = 0;
-	     i < tal_count(chain_events)
-	     || j < tal_count(channel_events)
-	     || k < tal_count(onchain_fees);
-	     /* Incrementing happens inside loop */) {
-		struct channel_event *chan;
-		struct chain_event *chain;
-		struct onchain_fee *fee;
-		u64 lowest = 0;
+	json_add_events(res, channel_events, chain_events, onchain_fees);
+	json_array_end(res);
+	return command_finished(cmd, res);
+}
 
-		if (i < tal_count(chain_events))
-			chain = chain_events[i];
-		else
-			chain = NULL;
-		if (j < tal_count(channel_events))
-			chan = channel_events[j];
-		else
-			chan = NULL;
-		if (k < tal_count(onchain_fees))
-			fee = onchain_fees[k];
-		else
-			fee = NULL;
+static struct command_result *param_outpoint(struct command *cmd,
+					     const char *name,
+					     const char *buffer,
+					     const jsmntok_t *tok,
+					     struct bitcoin_outpoint **outp)
+{
+	*outp = tal(cmd, struct bitcoin_outpoint);
+	if (json_to_outpoint(buffer, tok, *outp))
+		return NULL;
+	return command_fail_badparam(cmd, name, buffer, tok,
+				     "should be a txid:outnum");
+}
 
-		if (chain)
-			lowest = chain->timestamp;
+static struct command_result *json_edit_desc_utxo(struct command *cmd,
+						  const char *buf,
+						  const jsmntok_t *params)
+{
+	struct json_stream *res;
+	struct bitcoin_outpoint *outpoint;
+	const char *new_desc;
+	struct chain_event **chain_events;
 
-		if (chan
-		    && (lowest == 0 || lowest > chan->timestamp))
-			lowest = chan->timestamp;
+	if (!param(cmd, buf, params,
+		   p_req("identifier", param_outpoint, &outpoint),
```

Comment:
needs to be 'outpoint' to match the schema or it cant find the param, ill update in my commit soon to come

Reply:
resolved in https://github.com/niftynei/lightning/pull/9